src/Database/Driver.php

```diff
@@ -486,6 +519,16 @@ public function getMaxAliasLength(): ?int
         return static::MAX_ALIAS_LENGTH;
     }
 
+    /**
+     * Returns the number of connection retry attempts made.
+     *
+     * @return int
+     */
+    public function getLastConnectRetries(): int
```

Comment:
This could be just `getConnectRetries()`.

Reply:
Done. I also didn't update the retry count const name in SqlServer.